dashboard/test/models/unit_test.rb

```diff
@@ -2111,6 +2111,39 @@ class CanViewVersion < ActiveSupport::TestCase
     assert_equal "test-localized-title-default", unit.localized_title
   end
 
+  test 'next_unit returns next unit if there is another unit in unit group' do
+    unit_group = create :unit_group
+    unit1 = create :unit
+    unit2 = create :unit
+    create :unit_group_unit, unit_group: unit_group, script: unit1, position: 1
+    create :unit_group_unit, unit_group: unit_group, script: unit2, position: 2
+    unit1.reload
```

Comment:
Just for my own understanding, why does `unit1` have to be reloaded here but not in the subsequent tests?